plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanAParserBaseVisitor.java

```diff
@@ -0,0 +1,357 @@
+// ANTLR GENERATED CODE: DO NOT EDIT
+package org.elasticsearch.plan.a;
```

Comment:
So that IDEs work and other reasons. Its explained in the build if you take the time to read it.

Guys, please don't bikeshed stuff like "we should generate the antlr parser" unless you:
1. have been there and understand
2. plan on actually helping out to make it happen.

This is a very large piece of functionality and we got bigger shit to deal with, please don't waste time.